code/index.html

```diff
@@ -1,30 +1,110 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
 
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Project 3</title>
+  <!-- dont forget to add a css file and link it here! -->
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css?family=Sarpanch:400,500,700,900&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Bungee|Galindo|Righteous|Stalinist+One&display=swap"
+    rel="stylesheet">
+</head>
+
+<body>
+
+  <!-- video or image as a header is cool :) -->
+  <header>
+    <video width="100%" height="" autoplay loop muted poster="https://i.ytimg.com/vi/fOdeQPZptqY/hqdefault.jpg">
+      <source src="videos/gostya_iz_budushego_prekrasnoe_daleko_1985_fOdeQPZptqY_360p.mp4" type="video/mp4">
+      <source src="videos/SampleVideo_360x240_1mb.webm" type="video/webm">
+      <p>Your browser doesn't support HTML5 video.</p>
+    </video>
+
+    <article class="header-text">
+      <h1>Moscow Shuttle</h1>
+      <p>Fun times</p>
+    </article>
+  </header>
+
+  <main>
     <!-- Signup form -->
 
+    <section class="form">
+
+      <h1>Sign up!</h1>
```

Comment:
Try to use only one `<h1>` when it's possible. This heading would suit great as a `<h2>`